### ISSUE TITLE
Update extract_keywords_from_text behaviour to match rake-nltk

### ISSUE THREAD

**Issue by andrewrosss** [andrewrosss/rake-spacy#13]
As brought up in #12 it probably makes sense to have part of the API return a deduped version of the ranklist.

Specifically, the proposal is to have `Rake.extract_keywords_from_text` return a ranklist of type `List[Tuple[float, str]]`. This ranklist could then be reasonably deduped (as the default behaviour). The other similar method `Rake.apply` would continue to return a list of type `List[Tuple[float, Span]]`.

So in a nutshell the proposed change to the API would be:

```python
class Rake:

    def extract_keywords_from_text(self, text: str, dedup: bool = True) -> List[Tuple[float, str]]:
        ...

    def apply(self, text: str) -> List[Tuple[float, spacy.tokens.Span]]:
        ...
```
